.eslintrc

```diff
@@ -73,6 +73,24 @@
                 "max": 1
             }
         ],
-        "no-useless-concat": "error"
+        "no-useless-concat": "error",
+        "indent": ["warn", 4, {
+            "CallExpression": {
+                "arguments": 1
+            },
+            "SwitchCase": 1,
```

Comment:
SwitchCase cannot be set to "off" to disable linting about switch cases statements. I did not understand why. Someone proposed being able to set it to off [here]
(https://github.com/eslint/eslint/issues/7270) and developers said `it is good idea we won't be able to do it`. 

Similarly, in FunctionDecleration and FunctionExpression, "body" param is by default 1 and they cannot be set to "off" as well. If I set them to off, linter gives parsing error. I guess all of the three options are fine with the value "1". Body options are omitted since their default is 1. SwitchCase is not omitted and explicitly set to 1 since SwitchCase's default value is 0 and it is different from our current switch case indentation level which is 1. 

With this eslint configuration and last changes included, no linting errors left in all folders.

 Addressed your points in last review using "indent" rule. Plus, [there is one more](https://github.com/hazelcast/hazelcast-nodejs-client/pull/808/commits/053c3c4a7e7534a48ecbe534ea257d69fddf703c#diff-c13103958dde39fbec06f98103bd3387a6c127f5e3bc287d7a5fe42b4aaa3b53L77) which is due to 
```
{
    "FunctionExpression": {
         "body" : 1
     }
}
```
rule.

Reply:
I asked them in an issue see

https://github.com/eslint/eslint/issues/14167.

Good thing is we can actually configure that by ignoreNodes. I'll handle that

Reply:
After adding ignoreNode, I can now revert the change https://github.com/hazelcast/hazelcast-nodejs-client/pull/808/commits/053c3c4a7e7534a48ecbe534ea257d69fddf703c#diff-c13103958dde39fbec06f98103bd3387a6c127f5e3bc287d7a5fe42b4aaa3b53L77 which was due to implicit
```
{
    "FunctionExpression": {
         "body" : 1
     }
}
``` 
option of indent rule that I thought I could not disable.